nexus/tests/integration_tests/external_ips.rs

```diff
@@ -381,6 +382,50 @@ async fn test_floating_ip_create_name_in_use(
     );
 }
 
+#[nexus_test]
+async fn test_floating_ip_update(cptestctx: &ControlPlaneTestContext) {
+    let client = &cptestctx.external_client;
+
+    create_default_ip_pool(&client).await;
+    let project = create_project(client, PROJECT_NAME).await;
+
+    let fip = create_floating_ip(
+        client,
+        FIP_NAMES[0],
+        project.identity.name.as_str(),
+        None,
+        None,
+    )
+    .await;
+
+    let floating_ip_url = get_floating_ip_by_id_url(&fip.identity.id);
+
+    let new_fip_name: &str = "updated";
+    let new_fip_desc: &str = "updated description";
+
+    let updates: params::FloatingIpUpdate = params::FloatingIpUpdate {
+        identity: IdentityMetadataUpdateParams {
+            name: Some(String::from(new_fip_name).parse().unwrap()),
+            description: Some(String::from(new_fip_desc).parse().unwrap()),
+        },
+    };
+
+    let new_fip =
+        NexusRequest::object_put(client, &floating_ip_url, Some(&updates))
+            .authn_as(AuthnMode::PrivilegedUser)
+            .execute()
+            .await
+            .expect("")
+            .parsed_body::<FloatingIp>()
+            .unwrap();
```

Comment:
`object_put` is a little less noisy

https://github.com/oxidecomputer/omicron/blob/b993768e4d62515f33a058e4082dd7a721c9cee1/nexus/tests/integration_tests/ip_pools.rs#L157-L158